packages/server/src/connector/index.ts

```diff
@@ -188,6 +197,12 @@ async function subscribeEvents() {
       }
 
       if (event.method === 'Slash') {
+        if (
+          event.data[0].toString().toLowerCase() ===
+          settings.get().validatorId.toLowerCase()
```

Comment:
This will crash if validatorId is not set, either use ```settings.get().validatorId?.toLowerCase``` or get rid of .toLowerCase() altogether.